Makefile

```diff
@@ -63,7 +69,14 @@ test-all: babelify
 		--bail \
 		--reporter spec
 
+commit-dist:
+	make clean;
+	make;
```

Comment:
Yes, because `make clean` removes all compiled files into `dist/` folder. Compile and update source files is a recurrent task so I've created this `commit-dist` rule.